home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/service/impl/JwtService.java

```diff
@@ -0,0 +1,60 @@
+package com.softserveinc.ita.homeproject.homeservice.service.impl;
+
+import java.security.KeyPair;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.Date;
+
+import com.softserveinc.ita.homeproject.homeservice.service.TokenService;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.ExpiredJwtException;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.MalformedJwtException;
+import io.jsonwebtoken.SignatureAlgorithm;
+import io.jsonwebtoken.UnsupportedJwtException;
+import io.jsonwebtoken.security.Keys;
+import io.jsonwebtoken.security.SignatureException;
+import org.springframework.stereotype.Service;
+
+
+@Service
+public class JwtService implements TokenService {
```

Comment:
I think we a bit overload this functionality using jwt. The purpose of jwt is to deliver some information in it. What information is supposed to be delivered?

We can use a simple UIID generated token to confirm that invitation exists in the system.

Reply:
Okay